jgrapht-io/src/main/java/org/jgrapht/nio/gexf/GEXFAttributeType.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * (C) Copyright 2020-2020, by Dimitrios Michail and Contributors.
+ *
+ * JGraphT : a free Java graph-theory library
+ *
+ * See the CONTRIBUTORS.md file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the
+ * GNU Lesser General Public License v2.1 or later
+ * which is available at
+ * http://www.gnu.org/licenses/old-licenses/lgpl-2.1-standalone.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR LGPL-2.1-or-later
+ */
+package org.jgrapht.nio.gexf;
+
+/**
+ * Attribute types supported by GEXF.
+ * 
+ * @author Dimitrios Michail
+ */
+public enum GEXFAttributeType
+{
+    BOOLEAN("boolean"),
+    INTEGER("integer"),
+    LONG("long"),
+    FLOAT("float"),
+    DOUBLE("double"),
+    STRING("string"),
+    LISTSTRING("liststring"),
+    ANYURI("anyURI");
+
+    private String name;
+
+    private GEXFAttributeType(String name)
+    {
+        this.name = name;
+    }
+
+    /**
+     * Get a string representation of the attribute type
+     * 
+     * @return the string representation of the attribute type
+     */
+    public String toString()
+    {
+        return name;
+    }
+
+    /**
+     * Create a type from a string representation
+     * 
+     * @param value the name of the type
+     * @return the attribute type
+     */
+    public static GEXFAttributeType create(String value)
```

Comment:
Won't Enum.valueOf do this automatically?

Reply:
You are probably right. But then I will have to convert to uppercase first. Similar case with the toString() as well. If I remove it, I will have to use as toString().toLowercase()

Reply:
Fair enough.